apps/renderer/src/assets/main.css

```
@@ -189,4 +189,221 @@
+  left: -1px;
+  top: 14%;
+  bottom: 14%;
+  width: 3px;
+  border-radius: 2px;
+  background: linear-gradient(to bottom, transparent, var(--color-primary), transparent);
+  box-shadow: 0 0 10px var(--color-primary);
+  animation: _fx-edge-pulse 2.4s ease-in-out infinite;
+  pointer-events: none;
+}
+
+@keyframes _fx-edge-pulse {
```

Stylelint (17.12.0)

[error] Expected keyframe name "_fx-edge-pulse" to be kebab-case (keyframes-name-pattern)

(keyframes-name-pattern)